src/dashboard/src/components/filter/Filter.tsx

```diff
@@ -51,56 +49,57 @@ export const Filter: React.FC = () => {
   const operatingSystemItemId = currentParams.get('operatingSystemItem');
   const serverItemKey = currentParams.get('serverItem');
 
-  React.useEffect(() => {
-    // Create a key to unlock a dashboard update.
-    // A key requires all URL query parameters to be ready.
-    const tenant = tenantId ? tenants.find((t) => t.id === +tenantId) : undefined;
-    const organization = organizationId
-      ? organizations.find((t) => t.id === +organizationId)
-      : undefined;
-    const operatingSystemItem = operatingSystemItemId
-      ? operatingSystemItems.find((t) => t.id === +operatingSystemItemId)
-      : undefined;
-    const serverItem = serverItemKey
-      ? serverItems.find((t) => t.serviceNowKey === serverItemKey)
-      : undefined;
+  // React.useEffect(() => {
```

Comment:
This was causing some undesirable outcomes in the filter dropdowns.  I don't know if removing it will break other use-cases.  It was originally used for URL query parameter navigation, but that had to be removed because of other complications with annoying useEffect race conditions.